net/softethervpn/patches/series

```diff
@@ -0,0 +1,2 @@
+103-noeucjp.patch
+105-nosslv3.patch
```

Comment:
Please drop this file from - it is created on demand with `make package/softethervpn/{clean,prepare} QUILT=1`.